app/code/Magento/Catalog/Api/ProductRepositoryInterface.php

```diff
@@ -72,4 +72,11 @@ public function deleteById($sku);
      * @return \Magento\Catalog\Api\Data\ProductSearchResultsInterface
      */
     public function getList(\Magento\Framework\Api\SearchCriteriaInterface $searchCriteria);
+
+    /**
+     * Clean internal product cache
+     *
+     * @return void
+     */
+    public function cleanCache();
```

Comment:
it is not allowed change @api interfaces in patch releases. Unfortunately, we can not accept this change.